cockatrice/src/filtertree.cpp

```diff
@@ -272,16 +272,22 @@ bool FilterItem::acceptLoyalty(const CardInfoPtr info) const
     }
 }
 
-bool FilterItem::acceptPower(const CardInfoPtr info) const
+bool FilterItem::acceptPowerToughness(const CardInfoPtr info, CardFilter::Attr attr) const
 {
     int slash = info->getPowTough().indexOf("/");
-    return (slash != -1) ? (relationCheck(info->getPowTough().mid(0, slash).toInt())) : false;
-}
-
-bool FilterItem::acceptToughness(const CardInfoPtr info) const
-{
-    int slash = info->getPowTough().indexOf("/");
-    return (slash != -1) ? (relationCheck(info->getPowTough().mid(slash + 1).toInt())) : false;
+    if (slash == -1)
+        return false;
+    QString valueString;
+    if (attr == CardFilter::AttrPow)
```

Comment:
missing brackets </3

Reply:
right, sorry about that 😄 
BTW, would it be possible to modify clang-format to fix these type of mistakes?

Reply:
>BTW, would it be possible to modify clang-format to fix these type of mistakes?

clang-format can't since it's about modifying, not adding/removing stuff.
but clang-tidy could...
https://stackoverflow.com/questions/26111162/can-clang-format-add-braces-to-single-line-if-statements-etc

It has an option for [readability-braces-around-statements](http://clang.llvm.org/extra/clang-tidy/checks/readability-braces-around-statements.html).